File: Web/Khushi-Vaish_KhushiVaish_2024cs1147_2/Patterns/pattern5.cpp
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();

        int k = n / 2;
        int m = 1;
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= k; j++) {
                System.out.print("\t");
            }

            for (int j = 1; j <= m; j++) {
                System.out.print("*\t");
            }

            if (i <= n / 2) {
                k--;
                m += 2;
            } else {
                k++;
                m -= 2;
            }

            System.out.println();
        }

    }
}

File: AR-VR/saurabhcr007_saurabhchaudhary_1923csi1081_3/Pattern/pattern5.java
import java.util.*;

public class pattern5 {
    public static void main(String[] agrs){
        Scanner scn=new Scanner(System.in);
        int n=scn.nextInt();
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                if(i+j<n-1 || i>j){
                    System.out.print("   ");
                }
                else{
                    System.out.print("*  ");
                }
            }
            System.out.println("");
        }
    }
}
